plugin/auth_ldap/include/plugin_variables.h

```diff
@@ -0,0 +1,160 @@
+#ifndef _PLUGIN_VARIABLES_MPALDAP_H
+/* Copyright (c) 2019 Francisco Miguel Biete Banon. All rights reserved.
+
+   This program is free software; you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation; version 2 of the License.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program; if not, write to the Free Software Foundation,
+   51 Franklin Street, Suite 500, Boston, MA 02110-1335 USA */
+#define _PLUGIN_VARIABLES_MPALDAP_H
+
+#include "plugin/auth_ldap/include/plugin_common.h"
+#include "plugin/auth_ldap/include/pool.h"
+
+struct SYS_VAR;
+
+// If we define static variables in a header file we will have a copy of that
+// variable in each of the compilation units that include that header. Usually
+// not something we want, but perfect in this case to avoid duplicate variable
+// definitions for the simple and sasl plugins.
+
+// Static plugin variables
+static char *auth_method_name;
```

Comment:
It seems that variable `auth_method_name` is not used anywhere so it should probably be removed.